drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -233,8 +236,18 @@ def studio_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        # Do a bit of manipulation so we get the appearance of a list of zone options on
+        # items that still have just a single zone stored
+        data = copy.deepcopy(self.data)
```

Comment:
That's along the lines of what I suspected, but I wasn't totally sure. Thanks for checking!